### PR TITLE
Add rayshader_cheatsheet_3dviz.Rmd and rayshader_cheatsheet.pdf 

### PR DESCRIPTION
**Pull request by nys98**
Added Rmd file and pdf cheatsheet

### CODE REVIEW

File: rayshader_cheatsheet_3dviz.Rmd.Rmd [hunk 1]
```diff
@@ -0,0 +1,217 @@
+---
```
Comment (Jishan13) [inline rayshader_cheatsheet_3dviz.Rmd.Rmd:1]: The first line should start with a single hashtag #, followed by a single whitespace, and then the title.

File: rayshader_cheatsheet_3dviz.Rmd.Rmd [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+title: "Community Contribution"
+author: "Yu Song Ng"
+date: "2022-11-08"
+output: html_document
+---
+
+```{r setup, include=FALSE}
```
Comment (Jishan13) [inline rayshader_cheatsheet_3dviz.Rmd.Rmd:8]: use {r, include=FALSE} instead of {r setup, include=FALSE}.

**Review by Jishan13 — CHANGES_REQUESTED** [review]

File: rayshader_cheatsheet_3dviz.Rmd.Rmd (hunk 1 above)
Comment (nys98) [reply rayshader_cheatsheet_3dviz.Rmd.Rmd]: Have amended it, thank you!

File: rayshader_cheatsheet_3dviz.Rmd.Rmd (hunk 2 above)
Comment (nys98) [reply rayshader_cheatsheet_3dviz.Rmd.Rmd]: Amended this

Comment (Jishan13) [reply rayshader_cheatsheet_3dviz.Rmd.Rmd]: @nys98, your file says Rmd.Rmd. Can you please recommit with .Rmd

**Pull request merged**